conda_build/main_build.py

```diff
@@ -308,6 +309,8 @@ def execute(args, parser):
                     (conda_version[lang], version))
 
     if args.skip_existing:
+        if not isdir(config.bldpkgs_dir):
+            makedirs(config.bldpkgs_dir)
```

Comment:
Also, check `config.bldpkgs_dir` implementation: https://github.com/conda/conda-build/blob/master/conda_build/config.py#L109 (This causes the issues when `conda-bld/noarch` path is created, but `get_index` tries to reach `conda-bld/noarch` and `conda-bld/$PLATFORM` always, but fails only if `$PLATFORM` is not found: https://github.com/conda/conda/blob/master/conda/fetch.py#L124)